twilio_integration/overrides/notification.py

```diff
@@ -30,8 +30,9 @@ def send(self, doc):
 		super(SendNotification, self).send(doc)
 
 	def send_whatsapp_msg(self, doc, context):
-		send_whatsapp_message(
-			sender=self.twilio_number,
+		send_bulk_whatsapp_message(
```

Comment:
I guess we can go with `send_whatsapp_message` as a name, unless we have a different flow and single and bulk messages.